packages/flutter_tools/test/build_system/targets/assets_test.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/build_system/build_system.dart';
+import 'package:flutter_tools/src/build_system/targets/assets.dart';
+
+import '../../src/common.dart';
+import '../../src/testbed.dart';
+
+void main() {
+  group('copy_assets', () {
```

Comment:
Is there some general guidance for testing build rules that should be included in the API docs of the build system API?

Reply:
I added some comments on review/test responsibilities on [Target]